src/calculateRentalCost.js

```diff
@@ -5,6 +5,21 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const payPerDay = 40;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+
+  if (days < 3) {
```

Comment:
2. [checklist] Don't use literals in expressions (HARDCODE). Instead use constants explaining this values.

This refers to the number of days for the discount.